CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format located [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+- `sensu_auth_ldap` resource for ldap integration. (@webframp)
```

Comment:
I'd say either expand on this or add another item for the deprecation of `ad_servers`

Reply:
I'll add another item and push an update in just a few minutes

Reply:
Updated. For reference, I tried using https://docs.chef.io/dsl_custom_resource/#deprecate-and-alias instead but it immediately makes it a breaking change and I wanted to avoid that for now.

Reply:
It should not have made it an actual breaking change but thats probably because we are telling kitchen to behave in such a way: https://github.com/sensu/sensu-go-chef/blob/ce740244b0d4a37c3ac028e3e2923b37f5199068/.kitchen.yml#L7. We need to update that config if we are wanting to to use it here as that's mostly used when testing upgrades with external integrations. I honestly run my cookbooks with that as false except specifically when I look to support a new version. You can also always use the alias without a deprecation and give a `Log.warn ...`

Reply:
I can change it and use the DSL method if you prefer. I don't have a strong opinion about this one, but I didn't want to stray too much into changing the chef CI config again in an unrelated PR.

Reply:
I do prefer it because that is the one that has the least surprises to the community. This is the "established" way of handling this as far as I understand it. I personally only ever flip that to true when specifically testing versions to decide if we can support new/old versions on an upstream cookbook. I might run it on my own cookbooks more often but I am always targeting a specific version of chef with this and not latest. If we want to properly deprecate things there is a maintenance cost to do so. That being said I am not willing to die on this hill :wink:

Reply:
Updated here to use the DSL alias method